docs/developers.rst

```diff
@@ -421,6 +423,8 @@ Now at the bottom, under NetworkSettings, locate IP Address of your docker conta
 
 
 
+.. _already configured: http://www.python.org/
```

Comment:
@JustinPihony, why it is python.org?
Should that link point to some configuration example?

Reply:
Doh - copy and paste....thanks for the catch - will fix